hwc3-server/default/impl/HwcLoader.cpp

```diff
@@ -77,6 +78,8 @@ hwc2_device_t* HwcLoader::adaptGrallocModule(const hw_module_t* module) {
     int error = framebuffer_open(module, &device);
     if (error) {
         ALOGE("failed to open framebuffer device: %s", strerror(-error));
+        hw_device_t**dev = TO_HW_DEVICE_T_OPEN(&device);
```

Comment:
hw_device_t**dev should be "hw_device_t **dev"?

Reply:
#ifdef __cplusplus
#define TO_HW_DEVICE_T_OPEN(x) reinterpret_cast<struct hw_device_t**>(x)